mage_ai/frontend/pages/pipelines/[...slug].tsx

```diff
@@ -385,17 +385,21 @@ function PipelineDetailPage({
     },
   );
 
-  const [updatePipelineNoContent] = useMutation(api.pipelines.useUpdate(pipelineUUID));
-  const updatePipelineName = useCallback((name: string) => (
```

Comment:
Did removing this useCallback fix any existing bugs?

Reply:
This was me going back to refactor my `updatePipelineName` implementation, since I had done it in two different statements that could've been combined in one. I wasn't as familiar with the design pattern but I thought this was a good time to revise it